### PR TITLE
new pedals: great eastern, milkman amp, saturnworks looper, vox v810

### PR DESCRIPTION
**Pull request by jhpacker**
Added 4 new pedals:
great eastern fx co. small speaker overdrive
milkman the amp
saturnworks compact looper (horizontal)
vox v810 valvetone

### CODE REVIEW

**Review by cbserra — APPROVED** [review]

File: public/data/pedals.json [hunk 1]
```diff
@@ -10646,6 +10646,13 @@
 		"Height": 5.5,
 		"Image": "grace-design-felix.png"
 	},
+	{
+		"Brand": "Great Eastern FX Co.",
+		"Name": "Small Speaker Overdrive",
+		"Width": 2.5625
```
Comment (tehtrav) [inline public/data/pedals.json:10652]: missing comma

**Review by tehtrav — CHANGES_REQUESTED** [review]
Fix the one small spot in the JSON and I will approve once the build errors are resolved. Thanks!

**Comment by jhpacker** [comment]
thanks, sorry I missed that.

**Review by XlosVSM — APPROVED** [review]

**Review by ArislanShiva — CHANGES_REQUESTED** [review]
Hey sorry, I added the Milkman amp in my last PR from a recent request, didn't realize you had already made it.  You can remove yours now to clear the merge conflict.  Sorry!

**Pull request merged**